src/connectors/radcap.ts

```diff
@@ -0,0 +1,39 @@
+export {};
+
+function setupConnector() {
+  if (isDesktopPlayer()) {
+    setupDesktopPlayer();
+	} 
+  else {
+    setupMobilePlayer();
+	}
+}
+
+function isDesktopPlayer() {
+  return Boolean(document.querySelector('.radio'));
+}
+
+function setupDesktopPlayer() {
+  Connector.playerSelector = '.radio';
+
+  Connector.artistTrackSelector = '.stream.stream-signal > tbody > tr > td';
+
+  Connector.playButtonSelector = '#oframeplayer-d > pjsdiv:nth-child(7)';
+}
+
+function setupMobilePlayer() {
+  Connector.playerSelector = '.container';
+
+  Connector.artistTrackSelector = 'table.stream > tbody > tr > td';
+
+  const upperPlayerPlayButton = '#oframeaudioplayer2 > pjsdiv:nth-child(8) > pjsdiv:nth-child(2)';
+
+  const lowerPlayerPlayButton = '#oframeaudioplayer0 > pjsdiv:nth-child(8) > pjsdiv:nth-child(2)';
+
+  Connector.isPlaying = () => {
```

Comment:
Is this working for you? For me the play/pause is not working.

Reply:
You were right! I have just fixed it. Take a look, please.

Selectors are fine. Apparently, the problem is that they remove the play button with `visibility: hidden` instead of `display: none`, so `Util.isElementVisible` did not work in this case.